examples/test-rs/src/lib.rs

```diff
@@ -50,6 +53,15 @@ fn cache_miss() {
     );
 }
 
+#[spin_test]
```

Comment:
We're really starting to treat this example as a test suite now. I think we'll want to prioritize working on #42 to get away from this, but it's fine to leave this for now.

Reply:
Yep totally agree. I mostly left this here to demonstrate e2e what the work i was doing looks like. Will likely omit this when landing this PR.